appserver/admingui/corba/src/main/resources/sslEdit.jsf

```diff
@@ -65,7 +65,7 @@
         setPageSessionAttribute(key="createSslUrl", value="#{pageSession.parentUrl}/create-ssl");
         createMap(result="#{pageSession.createMap}");
         mapPut(map="#{pageSession.createMap}" key="target" value="#{pageSession.configName}");
-#include "/common/shared/sslPrepare.inc"
+#include "sslPrepare-copy.inc"
```

Comment:
Please just comment out the original include of sslPrepare.inc instead of removing it. 
and add a comment that this is workaround for fixing issue# 21716, and can be reverted once the REST issue xxxx is fixed.

Update issue# 21716 about workaround and the REST issue you have filed.